feconf.py

```diff
@@ -33,6 +33,10 @@
 IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
 
 # Whether we should serve the development or production experience.
+# This is environment specific variable(DEV_MODE) changes to true only
```

Comment:
Right, I get that from looking at the file. But what is the actual effect of DEV_MODE being set to True/False -- what does it affect?